static/lang/en.json

```diff
@@ -2061,6 +2061,8 @@
         "ItemNamePlaceholder": "Name",
         "ItemNewSkillVariant": "New Skill Variant",
         "ItemNotSupportedOnActor": "Item not supported on actor",
+        "ItemQuantityIncrease": "Click increases by 1, Shift-click increases by 5, Ctrl-click increases by 10",
+        "ItemQuantityDecrease": "Click decreases by 1, Shift-click decreases by 5, Ctrl-click decreases by 10",
```

Comment:
1. We try not to add top-level localization keys without a good reason. This can be added to "PF2E.Actor.Inventory".
2. Use language seen in "PF2E.DamageButton", joining by `&#013;`